src/aleph/storage.py

```diff
@@ -145,7 +146,8 @@ async def verify_content_hash(
     Checks that the hash of a content we fetched from the network matches the expected hash.
     :return: True if the hashes match, False otherwise.
     """
-    ipfs_enabled = app["config"].ipfs.enabled.value
+    config = get_config()
+    ipfs_enabled = config.ipfs.enabled.value
```

Comment:
Why not `get_config().ipfs.enabled.value` ?

Reply:
While coding, my train of thought was 1) get the config and 2) use it where necessary (there are several uses of the config object in most cases). Your suggestion could be useful once we have a dynamic configuration system, reloadable at runtime. Until we have that, I'd keep the local variable.